include/swift/AST/IRGenOptions.h

```diff
@@ -259,6 +259,8 @@ class IRGenOptions {
     unsigned Hash = (unsigned)OptMode;
     Hash = (Hash << 1) | DisableLLVMOptzns;
     Hash = (Hash << 1) | DisableSwiftSpecificLLVMOptzns;
+    Hash = (Hash << 1) | GenerateProfile;
+    Hash = (Hash << 1) | Sanitizers.contains(SanitizerKind::Fuzzer);
```

Comment:
Honestly we might as well mix in the whole thing. It's not just about empty files; it's also about *non-*empty files that only change due to LLVM passes, and I'm not sure I trust that all the other sanitizers don't need this.